js/apps/admin-ui/src/realm-settings/user-profile/AttributesTab.tsx

```diff
@@ -86,9 +86,15 @@ export const AttributesTab = ({ setTableData }: AttributesTabProps) => {
       if (!config?.attributes) return;
 
       const translationsToDelete = config.attributes.find(
-        (attribute) => attribute.name === attributeToDelete,
+        (attribute: UserProfileAttribute) =>
+          attribute.name === attributeToDelete,
       )?.displayName;
 
+      const formattedTranslationsToDelete = translationsToDelete?.substring(
```

Comment:
Could you add a comment here with some explanation as to why we need to slice this string like this?